test/assets/fixtures/k8s-lonely-pod.json

```diff
@@ -0,0 +1,28 @@
+{
+    "kind": "List",
```

Comment:
yeah its a generic list of things, each item specifies its type.  I ran all of these with `osc create -f whatever.json`